admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
+
+=== Builds on a Virtual Network are Failing
+
+If you are installing OpenShift using a virtual network (for example,
```

Comment:
@bfallonf Pretty sure we talked about it in a weekly mtg or something. It made its way into the guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#product-name-version-attributes